superset-frontend/package.json

```diff
@@ -112,6 +112,7 @@
     "geolib": "^2.0.24",
     "global-box": "^1.2.0",
     "html-webpack-plugin": "^4.5.1",
+    "immer": "^8.0.1",
```

Comment:
When adding a new library, it would be nice to add a note in PR description to explain why it is needed. Sometimes it's not very obvious what the purpose of the library is.